src/neuvector/chart/templates/peerauthentication/neuvector-controller-pa.yaml

```diff
@@ -12,7 +12,7 @@ spec:
     matchLabels:
       app: neuvector-controller-pod
   mtls:
-    mode: STRICT
+    mode: PERMISSIVE
```

Comment:
I missed this change earlier - what is the behavior with this set to STRICT? The way this behaves it would allow PERMISSIVE on all ports for the controller, I'd hope we could instead just set some ports to PERMISSIVE such as the two below here.

Reply:
I'll test this out and this will be a separate PR when we enable ambient. Going to revert for this PR